### PR TITLE
BO-2273: add and manage volume prices for offers

### PR DESCRIPTION
**Pull request by tobi812**
## PR Description

Integration guide for adding and managing volume prices for offers.

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-merchant-portal-product-offer-management-feature-integration.md [hunk 1]
```diff
@@ -94,14 +95,66 @@ class SalesMerchantPortalGuiDependencyProvider extends SprykerSalesMerchantPorta
 Make sure that the `ProductOfferMerchantOrderItemTableExpanderPlugin` is set up by opening `http://zed.mysprykershop.com/sales-merchant-portal-gui/orders`. Click on any of the orders and check that the *Merchant Reference* and *Product Offer SKU* are present.
 
 {% endinfo_block %}
+    
+#### Extend and validate PriceProducts with volume quantity for Offers
 
-### Add the Offer widget to MerchantDashobard
+Activate the following plugins:
```
Comment (ilyakubanov) [inline docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-merchant-portal-product-offer-management-feature-integration.md:101]: all this changes should be in Marketplace Product Offer Prices feature integration

File: docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-merchant-portal-product-offer-management-feature-integration.md [hunk 2]
```diff
@@ -131,3 +184,44 @@ class DashboardMerchantPortalGuiDependencyProvider extends SprykerDashboardMerch
 Make sure that the `OffersMerchantDashboardCardPlugin` plugin is set up by opening `http://zed.mysprykershop.com/dashboard-portal-gui`. The Offers widget should show up on the page.
 
 {% endinfo_block %}
+
+### 4) Configure export to Redis
```
Comment (ilyakubanov) [inline docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-merchant-portal-product-offer-management-feature-integration.md:188]: the same

**Review by ilyakubanov — COMMENTED** [review]
Please also recheck Marketplace Product Offer Prices feature IG:
- some transfer properties as PriceProductFilter.Filter, PriceProductFilter.value don't exist

**Review by ilyakubanov — DISMISSED** [review]

File: docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-product-offer-prices-feature-integration.md [hunk 3]
```diff
@@ -92,8 +92,6 @@ Make sure that the following changes were applied in transfer objects:
 | PriceProductOfferCriteria | class | created | src/Generated/Shared/Transfer/PriceProductOfferCriteriaTransfer |
 | DataImporterReaderConfiguration | class | created | src/Generated/Shared/Transfer/DataImporterReaderConfigurationTransfer |
 | Currency.Code | attribute | created | src/Generated/Shared/Transfer/CurrencyTransfer |
-| PriceProductFilter.Filter | attribute | created | src/Generated/Shared/Transfer/PriceProductFilterTransfer |
```
Comment (sakharova-yuliia) [inline docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-product-offer-prices-feature-integration.md:95]: Do we have to delete them? Propose just to fix attribute names, cos we're registering 
 `PriceProductFilter.productOfferReference` and `PriceProductFilter.quantity` transfer properties in PriceProductOffer and PriceProductOfferVolume accordingly

Comment (ilyakubanov) [reply docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-product-offer-prices-feature-integration.md:95]: Please add PriceProductFilter.productOfferReference (PriceProductFilter.quantity exists in Prices feature)

**Review by gibmirmilch — APPROVED** [review]

**Pull request merged**